pkg/types/ccipocr3/plugin_execute_types.go

```diff
@@ -11,3 +17,35 @@ type ExecutePluginReportSingleChain struct {
 	Proofs              []Bytes32     `json:"proofs"`
 	ProofFlagBits       BigInt        `json:"proofFlagBits"`
 }
+
+type CommitReportWithMessages struct {
```

Comment:
Where's this used?

Reply:
Oops, didn't end up using this. It might still be needed, but I'll remove it for this PR.